utils/scripts/install-dev.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This script sets up a new development node with all dependencies needed to
+# build DAOS from source.
+
+# Requirements:
+# - Run as a non-root user with sudo access.
+
+if [ "$(whoami)" == "root" ]; then
+	echo "Run this script as the non-root user you will use to build DAOS"
+	exit 1
+fi
+
+distro=$1
+case "$distro" in
+	centos7 | el8 | leap15 | ubuntu)
+		echo "Setting up node for $distro...";;
+
+	*)
+		echo "Invalid distro option: \"$distro\""
+		echo "syntax: $0 <el8|leap15|ubuntu|centos7>"
+		exit 1
+esac
+
+# Install dependencies from the OS repos.
+SCRIPT_DIR="$(dirname "$(realpath -e "$BASH_SOURCE")")"
```

Comment:
(lint) Expanding an array without an index only gives the first element. [SC2128]